cas-server-webapp-throttle/src/main/java/org/jasig/cas/web/support/AbstractThrottledSubmissionHandlerInterceptorAdapter.java

```diff
@@ -85,8 +86,8 @@ public final void postHandle(final HttpServletRequest request, final HttpServlet
             return;
         }
 
-        // User successfully authenticated
-        if (SUCCESSFUL_AUTHENTICATION_EVENT.equals(context.getCurrentEvent().getId())) {
+        MutableAttributeMap<Object> flowScope = context.getFlowScope();
```

Comment:
Wouldn't it be simpler to make sure the current event id is appropriately returned instead, rather than modifying the webflow directly as such?